File: Exercicio revisão/Exercicio revisão/pessoa.py
class pessoa:
    def __init__(self, origem, idade, genero, cor, profissao):
        self.origem = origem
        self.idade = idade
        self.genero = genero
        self.cor = cor
        self.profissao = profissao

    def apresentacao(self):
        return (f"Olá, Sou do {self.origem}, tenho {self.idade} anos, "
                f"meu gênero é {self.genero}, minha etinia é {self.cor} e trabalho como {self.profissao}.")

pessoa1 = pessoa("Rio de Janeiro", 21, "masculino", "branco", "professor")
print(pessoa1.apresentacao())

File: Exercicio revisão/Exercicio revisão/pessoa.java
public class pessoa {
    private String origem;
    private int idade;
    private String genero;
    private String cor;
    private String profissao;


    public pessoa(String origem, int idade, String genero, String cor, String profissao) {
        this.origem = origem;
        this.idade = idade;
        this.genero = genero;
        this.cor = cor;
        this.profissao = profissao;
    }

    public String apresentacao() {
        return "Olá, sou do " + origem + ", tenho " + idade + " anos, "
                + "meu gênero é " + genero + ", minha etinía é " + cor + " e trabalho como " + profissao + ".";
    }

    public static void main(String[] args) {
        pessoa pessoa1 = new pessoa("Rio de Janeiro", 21, "masculino", "branco", "professor");
        System.out.println(pessoa1.apresentacao());
    }
}
